lib/install/management/create.go

```diff
@@ -211,3 +213,72 @@ func (d *Dispatcher) uploadImages(files map[string]string) error {
 	}
 	return nil
 }
+
+// cleanupAfterCreationFailed cleans up the dangling resource pool for the failed VCH and any bridge network if there is any.
+// The function will not abort and early terminate upon any error during cleanup process. Error details are logged.
+func (d *Dispatcher) cleanupAfterCreationFailed(conf *config.VirtualContainerHostConfigSpec, cleanupNetwork bool) {
+	defer trace.End(trace.Begin(conf.Name, d.op))
+	var err error
+
+	d.op.Info("VCH creation has failed in previous steps, start cleaning up dangling resources...")
```

Comment:
General message structure comments:
* use of `...` is misleading unless you're going to append `done`, `failed`, `complete` or similar to the same line once the operation is completed given traditional console usage.
* it's ok to be imperative and concise, e.g. `Cleaning up after failure`
* multi-line output in this manner should only be used if it adds significant value as it's prone to tearing.

I don't think any of this needs to be at Info level, debug is fine - it will not take significant time under normal conditions and it's not something that requires user intervention. At most one info message noting cleanup is occurring but even that I consider optional.